### PR TITLE
T1 staticlanding

### PR DESCRIPTION
**Pull request by erreinoso**
I did the html and css of the landing and filter options. Please, check toggles, there is something that doesn't work properly.

### CODE REVIEW

File: client/src/components/Filter.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          <label for="switch"></label><p>animales famosos</p></div>
+        <div className="App--filter__option hidden">
+          <input type="checkbox" className="switch" id="switch" />
+          <label for="switch"></label><p>nombres compuestos</p></div>
+        <div className="App--filter__option hidden">
+          <input type="checkbox" className="switch" id="switch" />
+          <label for="switch"></label><p>épicos</p></div>
+        <div className="App--filter__option hidden">
+          <input type="checkbox" className="switch" id="switch" />
+          <label for="switch"></label><p>clásicos</p></div>
```
Comment (FCalabria) [inline client/src/components/Filter.js:28]: I'm sure this can be done with a loop instead of repeating so much code.

File: client/src/index.js [hunk 2]
```diff
@@ -1,21 +1,16 @@
...
-import rootReducer from './store/reducers'
-
+import rootReducer from './store/reducers';
 const store = createStore(rootReducer)
 
 ReactDOM.render(
-  <React.StrictMode>
-      <Provider store={store}>
-        <App />
-      </Provider>,
```
Comment (FCalabria) [inline client/src/index.js:17]: If you dont want to set up redux for now (which is perfectly fine), you can also delete all the related imports

**Review by FCalabria — COMMENTED** [review]
I'm not even going to look at the css... why are you not using Tailwind? If you don't like it you could have tell before :(

File: client/src/components/Filter.js (hunk 1 above)
Comment (erreinoso) [reply client/src/components/Filter.js:28]: you are right!

Comment (erreinoso) [reply client/src/components/Filter.js:28]: we checked that SASSand Tailwind could work together and we installed it. Then I forgot to use it   :| my bad! i will improve it

Comment (FCalabria) [reply client/src/components/Filter.js:28]: Perfect, but consider why do you need sass. Maybe it's not necesary ;)

Comment (claudiasanes) [reply client/src/components/Filter.js:28]: okey perfect, I'm gonna try to make a loop with it :)

**Review by claudiasanes — APPROVED** [review]

**Comment by claudiasanes** [comment]
Great, Elena! I'm going to make some tiny changes in the CSS, try to fix the filter JS and the filters loop, let's see if I still remember anything of how to work with React!

**Pull request merged**